docs/aggregation.md

```diff
@@ -1,12 +1,34 @@
-# qgrid 
+# qgrid
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows='data'>
+	<q-grid:columns>
+		<q-grid:column key="Name" title="Name" aggregation="count"></q-grid:column>
+		<q-grid:column key="Age" title="Age" aggregation="avg"></q-grid:column>
+		<!-- max || min || sum || count || avg -->
+	</q-grid:columns>
+	<!-- naming: aggregation, aggregation-footer -->
+	<q-grid:aggregation></q-grid:aggregation>
```

Comment:
right now we have for custom aggregation, one of the reasons to do like that - agg should be directly under the column values
```html
<q-grid columns="$ctrl.columns"
		  rows="$ctrl.rows">
	<q-grid:columns>
		<q-grid:column key="gender">
			<q-grid:template for="foot" let="$ctrl">
				<b>male: {{100*$ctrl.getCountGender($view.rows, 'male')/$view.rows.length}}%</b> <br/>
				<b>female: {{100*$ctrl.getCountGender($view.rows, 'female')/$view.rows.length}}%</b>
			</q-grid:template>
		</q-grid:column>
	</q-grid:columns>
</q-grid>
```